nilearn/datasets/struct.py

```diff
@@ -106,38 +114,134 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
-def load_mni152_template():
-    """Load skullstripped 2mm version of the MNI152 originally distributed
-    with FSL.
+def load_mni152_template(resolution=2):
+    """This function takes the skullstripped 1mm-resolution version of the
+    MNI152 T1 template, originally distributed with FSL, and re-samples it
+    using a different resolution, if specified.
 
     For more information, see :footcite:`FONOV2011313`,
     and :footcite:`Fonov2009`.
 
+    Parameters
+    ----------
+    resolution: int, optional, Default = 2
+        If resolution is different from 1, the template is re-sampled with the
+        specified resolution.
+
+        .. versionadded:: 0.8.1
+
     Returns
     -------
-    mni152_template : nibabel object corresponding to the template
+    mni152_template : Nifti1Image, image representing the re-sampled
+        whole-brain template
 
     References
     ----------
     .. footbibliography::
 
     """
-    return check_niimg(MNI152_FILE_PATH)
 
+    brain_template = check_niimg(MNI152_FILE_PATH)
+    new_brain_template = new_img_like(
+        brain_template, get_data(brain_template).astype('uint16'))
+
+    # Change the resolution of the template
+    if resolution != 1:
+        new_brain_template = resampling.resample_img(new_brain_template,
+                                                     np.eye(3) * resolution)
+
+    return new_brain_template
+
+
+def load_mni152_gm_template(resolution=2):
```

Comment:
We need to add this function to `doc/modules/reference.rst` to include it in the docs.